resources/sass/components/_map-aside.scss

```diff
@@ -3,7 +3,16 @@
     padding: 0 $indent-m;
     height: calc(100vh - 112px);
     overflow-y: scroll;
+    @media screen and (max-width: 1040px) {
```

Comment:
`1040px` has to be a variable since its used in other places

Reply:
id added

Reply:
is added